.travis.yml

```diff
@@ -43,6 +43,6 @@ script:
 after_success:
   - if [[ "$TRAVIS_PULL_REQUEST" == "false" && "$TRAVIS_BRANCH" == "appsembler/hawthorn/master" ]]; then
       docker login -u $DOCKERUSER -p $DOCKERPASSWORD ;
-      travis_wait 50 docker build -f docker/build/edxapp/Dockerfile . -t appsembler/edxapp:latest ;
-      docker push appsembler/edxapp:latest ;
+      travis_wait 50 docker build -f docker/build/edxapp/Dockerfile . -t appsembler/edxapp:hawthorn ;
```

Comment:
This change is now removed in Juniper. It won't have effect. We need to fix it via GitHub actions.